bokeh/__init__.py

```diff
@@ -1,6 +1,7 @@
 from __future__ import absolute_import, print_function
 
 import logging
+import sys
```

Comment:
Ah, thanks for pointing that out!  I had left that in for testing purposes initially and had forgotten it.  I'll remember to run pyflakes on files changed in the future.